src/MIDebugEngine/Engine.Impl/DebuggedProcess.cs

```diff
@@ -777,7 +777,7 @@ private void CheckCygwin(List<LaunchCommand> commands, LocalLaunchOptions localL
 
         private void AddExecutablePathCommand(IList<LaunchCommand> commands)
         {
-            string exe = EscapePath(_launchOptions.ExePath);
+            string exe = this.EscapePath(_launchOptions.ExePath);
```

Comment:
>EscapePath [](start = 30, length = 10)

hmm... as I think about it more, I think this isn't complicated enough :). In Android gdb server scenarios at least, I believe this and the working directory and both local (Windows) paths.

Is this how we tell GDB the exe path in GDB Server launch scenarios?